courses/pydata/info.yml

```diff
@@ -0,0 +1,21 @@
+title: Kurz datové analýzy
```

Comment:
Nazýváme to vlastně "datovou analýzou" / "analytikou"? Nechceme něco nadřazeného?

Reply:
Za mě je datová analýza či analýza dat pojem dostatečně široký. https://cs.wikipedia.org/wiki/Anal%C3%BDza_dat